### PR TITLE
[Assessor] added ability to download original application PDF

### PR DESCRIPTION
**Pull request by rusllonrails**
[Trello Story](https://trello.com/c/TS7BZGtA/243-display-a-button-for-admins-to-download-the-original-pdf-of-the-application-at-the-deadline)

```
Assessors should also be able to download the orginal pdf
of the application at the deadline
```

### CODE REVIEW

**Comment by rusllonrails** [comment]
<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;TS7BZGtA&#x2F;243-display-a-button-for-admins-to-download-the-original-pdf-of-the-application-at-the-deadline"> Display a button for admins to download the original pdf of the application at the deadline</a>

File: app/policies/form_answer_policy.rb [hunk 1]
```diff
@@ -77,10 +77,10 @@ def can_see_corp_responsibility?
 
   def can_review_corp_responsibility?
     can_see_corp_responsibility? &&
-    (admin? || subject.lead?(record) || subject.primary?(record))
+    can_update_by_admin_lead_and_primary_assessors?
```
Comment (eritiro) [inline app/policies/form_answer_policy.rb]: I like it

File: app/views/admin/form_answers/docs/_download_original_application_pdf_before_deadline.html.slim [hunk 2]
```diff
@@ -1,5 +1,8 @@
 - if policy(@form_answer).can_download_original_pdf_of_application_before_deadline?
-  = link_to original_pdf_before_deadline_admin_form_answer_url(@form_answer, format: :pdf),
+
+  - target_url = pundit_user.is_a?(Admin) ? original_pdf_before_deadline_admin_form_answer_url(@form_answer, format: :pdf) : original_pdf_before_deadline_assessor_form_answer_url(@form_answer, format: :pdf)
+
+  = link_to target_url,
```
Comment (eritiro) [inline app/views/admin/form_answers/docs/_download_original_application_pdf_before_deadline.html.slim]: maybe this should be in a helper method that calculates the target_url and outputs the link

**Comment by eritiro** [comment]
looks good!

File: app/views/admin/form_answers/docs/_download_original_application_pdf_before_deadline.html.slim (hunk 2 above)
Comment (rusllonrails) [inline app/views/admin/form_answers/docs/_download_original_application_pdf_before_deadline.html.slim]: maybe this should be in a helper method that calculates the target_url and outputs the link
=> yeah, that make sense. Gonna fix

**Pull request merged**